From 5a2ae894a20b3a4009466a194fe105bf399309ea Mon Sep 17 00:00:00 2001
From: harusame_0616 <kakino1999@gmail.com>
Date: Wed, 14 Jul 2021 12:03:43 +0900
Subject: [PATCH] no message

---
 Project1/WinMain.cpp | 35 +----------------------------------
 1 file changed, 1 insertion(+), 34 deletions(-)

diff --git a/Project1/WinMain.cpp b/Project1/WinMain.cpp
index 1e93071..02c393f 100644
--- a/Project1/WinMain.cpp
+++ b/Project1/WinMain.cpp
@@ -76,17 +76,10 @@ namespace {
 	ComPtr<ID3D12CommandQueue> command_queue = nullptr;
 
 
-<<<<<<< HEAD
 	//ãƒ¬ãƒ³ãƒ€ãƒ¼ã‚¿ãƒ¼ã‚²ãƒƒãƒˆã®è¨­å®š(Descriptor)ã‚’VRAMã«ä¿å­˜ã™ã‚‹ãŸã‚å¿…è¦
 	ComPtr<ID3D12DescriptorHeap> rtv_descriptor_heap = nullptr;
 	//ä¸Šã®ãƒ¬ãƒ³ãƒ€ãƒ¼ã‚¿ãƒ¼ã‚²ãƒƒãƒˆãƒ‡ã‚¹ã‚¯ãƒªãƒ—ã‚¿ã®1å€‹å½“ãŸã‚Šã®ã‚µã‚¤ã‚ºã‚’è¦šãˆã‚‹
 	UINT rtv_descriptor_size = 0;
-=======
-	//ƒŒƒ“ƒ_[ƒ^[ƒQƒbƒg‚Ìİ’è(Descriptor)‚ğVRAM‚É•Û‘¶‚·‚é‚½‚ß•K—v
-	ComPtr<ID3D12DescriptorHeap> rtv_descriptr_heap = nullptr;
-	//ã‚ÌƒŒƒ“ƒ_[ƒ^[ƒQƒbƒgƒfƒXƒNƒŠƒvƒ^‚Ì1ŒÂ“–‚½‚è‚ÌƒTƒCƒY‚ğŠo‚¦‚é
-	UINT rtv_descriptr_size = 0;
->>>>>>> 013d0d23e9813dd8d15917b9c871245ea105a894
 
 	//GPUå‡¦ç†ãŒçµ‚ã‚ã£ã¦ã„ã‚‹ã‹ã‚’ç¢ºèªã™ã‚‹ãŸã‚ã«å¿…è¦ãªãƒ•ã‚§ãƒ³ã‚¹ã‚ªãƒ–ã‚¸ã‚§ã‚¯ãƒˆ
 	ComPtr<ID3D12Fence> d3d12_fence = nullptr;
@@ -109,7 +102,6 @@ namespace {
 	//ç¾åœ¨ã®ãƒãƒƒã‚¯ãƒãƒƒãƒ•ã‚¡ã®ã‚¤ãƒ³ãƒ‡ãƒƒã‚¯ã‚¹
 	int backbuffer_index = 0;
 
-<<<<<<< HEAD
 	/*-------------------------------------------*/
 	/*  ãƒ—ãƒ­ãƒˆã‚¿ã‚¤ãƒ—å®£è¨€                                   */
 	/*-------------------------------------------*/
@@ -130,16 +122,6 @@ namespace {
 	/// @param wParam ãƒ¡ãƒƒã‚»ãƒ¼ã‚¸ã®ãƒ‘ãƒ©ãƒ¡ãƒ¼ã‚¿
 	/// @param lParam ã€€ãƒ¡ãƒƒã‚»ãƒ¼ã‚¸ã®ãƒ‘ãƒ©ãƒ¡ãƒ¼ã‚¿
 	/// @return ãƒ¡ãƒƒã‚»ãƒ¼ã‚¸ã®æˆ»ã‚Šå€¤
-=======
-
-
-	/// @brief  ƒEƒBƒ“ƒhƒEƒvƒƒV[ƒWƒƒ
-	/// @param hwnd  ƒEƒBƒ“ƒhƒEƒnƒ“ƒhƒ‹
-	/// @param message ƒƒbƒZ[ƒW‚Ìí—Ş
-	/// @param wParam ƒƒbƒZ[ƒW‚Ìƒpƒ‰ƒ[ƒ^
-	/// @param lParam @ƒƒbƒZ[ƒW‚Ìƒpƒ‰ƒ[ƒ^
-	/// @return ƒƒbƒZ[ƒW‚Ì–ß‚è’l
->>>>>>> 013d0d23e9813dd8d15917b9c871245ea105a894
 	LRESULT CALLBACK WndProc(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
 	{
 		switch (message)
@@ -311,12 +293,7 @@ namespace {
 				return false;
 			}
 
-<<<<<<< HEAD
 			//IDXGISwapChain1::Asé–¢æ•°ã‹ã‚‰IDXGISwapChain4ã‚¤ãƒ³ã‚¿ãƒ¼ãƒ•ã‚§ãƒ¼ã‚¹ã‚’å–å¾—
-=======
-			//IDXGISwapChain1::AsŠÖ”‚©‚çIDXGISwapChain4ƒCƒ“ƒ^[ƒtƒF[ƒX‚ğæ“¾
-			
->>>>>>> 013d0d23e9813dd8d15917b9c871245ea105a894
 			hr = sc.As(&swap_chain);
 			if (FAILED(hr))
 			{
@@ -328,7 +305,6 @@ namespace {
 		{
 			//ãƒ¬ãƒ³ãƒ€ãƒ¼ã‚¿ãƒ¼ã‚²ãƒƒãƒˆç”¨ãƒ‡ã‚¹ã‚¯ãƒªãƒ—ã‚¿ãƒ’ãƒ¼ãƒ—ã®è¨­å®šæ§‹é€ ä½“
 			D3D12_DESCRIPTOR_HEAP_DESC desc{};
-<<<<<<< HEAD
 			desc.Type = D3D12_DESCRIPTOR_HEAP_TYPE_RTV;
 			desc.NumDescriptors = backbuffer_size;
 			desc.Flags = D3D12_DESCRIPTOR_HEAP_FLAG_NONE;
@@ -458,8 +434,6 @@ namespace {
 			{
 				return false;
 			}
-=======
->>>>>>> 013d0d23e9813dd8d15917b9c871245ea105a894
 		}
 		return true;
 	}
@@ -467,7 +441,6 @@ namespace {
 	/// @brief  D3Dã‚ªãƒ–ã‚¸ã‚§ã‚¯ãƒˆè§£æ”¾å‡¦ç†
 	void FinalizeD3D()
 	{
-<<<<<<< HEAD
 		WaitForGpu();
 	}
 
@@ -631,9 +604,6 @@ namespace {
 
 		//æ¬¡ã®ãƒ•ãƒ¬ãƒ¼ãƒ ã®ãŸã‚ã«ãƒ•ã‚§ãƒ³ã‚¹å€¤æ›´æ–°
 		fence_values[backbuffer_index] = current_value + 1;
-=======
-
->>>>>>> 013d0d23e9813dd8d15917b9c871245ea105a894
 	}
 }// namespace
 
@@ -735,16 +705,13 @@ int WINAPI wWinMain(_In_ HINSTANCE hInstance,
 		}
 		else
 		{
-<<<<<<< HEAD
 			//ã“ã“ã«ã‚²ãƒ¼ãƒ ã®å‡¦ç†ã‚’æ›¸ã
 			Render();
-=======
-			//‚±‚±‚ÉƒQ[ƒ€‚Ìˆ—‚ğ‘‚­
->>>>>>> 013d0d23e9813dd8d15917b9c871245ea105a894
 		}
 	}
 
 	FinalizeD3D();
+	CoUninitialize();
 
 	return 0;
 }
\ No newline at end of file
-- 
2.28.0.windows.1

